visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -1,21 +1,70 @@
 import tensorflow as tf
 
 # TODO: Replace with custom model variant
-model = tf.keras.applications.MobileNetV2(input_shape=None,
-                                          include_top=True,
-                                          weights="imagenet")
-
+#model = tf.keras.applications.MobileNetV2(input_shape=None,
+#                                          include_top=True,
+#                                          weights="imagenet")
+model = tf.keras.applications.Xception( include_top=True,
```

Comment:
We did quite a bit of research prior to selecting MobileNet as our model's architecture. I would like to discuss/understand this in more detail.